pedpy/methods/method_utils.py

```diff
@@ -238,20 +238,20 @@ def compute_time_distance_line(
 
     Compute the time (in frames) and distance to the first crossing of the
     measurement line. For further information how the crossing frames are
-    computed see :obj:`~compute_crossing_frames`. All frames after a
+    computed see :func:`~compute_crossing_frames`. All frames after a
     pedestrian has crossed the line will be omitted in the results.
 
     Args:
         traj_data (TrajectoryData): trajectory data
         measurement_line (MeasurementLine): line which is crossed
 
-    Returns: DataFrame containing 'ID', 'frame', 'time' (frames to crossing),
-        and 'distance' (to measurement line)
+    Returns: DataFrame containing 'ID', 'frame', 'time_till_crossing' (seconds
```

Comment:
no colloquialism 'till', either use until or be more explicit e.g. 'crossing_at' which again is seconds since frame 0